frontend/pulse3d/src/pages/_app.js

```diff
@@ -28,7 +28,7 @@ export const AuthContext = createContext();
 
 const availablePages = {
   user: ["/uploads", "/upload-form", "/account"],
-  admin: ["/uploads", "/new-user"],
+  admin: ["/uploads", "/new-user", "/users-info"],
```

Comment:
We could probably just name this page `/users`. It hits a different server so it wouldn't collide with `/users` routes that we create in the BE